src/PrestaShopBundle/Form/Admin/Type/TranslatableType.php

```diff
@@ -44,12 +45,37 @@ class TranslatableType extends AbstractType
      */
     private $locales;
 
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    /**
+     * @var bool indicates whether to save the selected form language or not
+     */
+    private $saveFormLocaleChoice;
+
+    /**
+     * @var string default form language ID
+     */
+    private $defaultFormLanguageId;
+
     /**
      * @param array $locales
+     * @param UrlGeneratorInterface $urlGenerator
+     * @param bool $saveFormLocaleChoice
+     * @param int $defaultFormLanguageId
      */
-    public function __construct(array $locales)
-    {
+    public function __construct(
```

Comment:
We can modify the constructor because this class has been introduced for 1.7.6 👍 https://github.com/PrestaShop/PrestaShop/pull/10704